astropy/cosmology/utils.py

```diff
@@ -1,14 +1,21 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-__all__ = []  # nothing is publicly scoped
+from __future__ import annotations
```

Comment:
You probably know better than me, but from what I've seen `annotations` ["may not be the future"](https://docs.python.org/3.11/whatsnew/3.11.html#pep-563-may-not-be-the-future). PEP 563 has been delayed twice in 3.10 and 3.11. Or maybe three times since it doesn't appear to be in 3.12. 

So should we be using this in our code?